src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2566,6 +2566,13 @@ public sealed partial class WeakReference<T> : System.Runtime.Serialization.ISer
         public void SetTarget(T target) { }
         public bool TryGetTarget(out T target) { target = default(T); return default(bool); }
     }
+    public partial class UnhandledExceptionEventArgs : System.EventArgs
+    {
+        public UnhandledExceptionEventArgs(object exception, bool isTerminating) { }
+        public object ExceptionObject { get { throw null; } }
+        public bool IsTerminating { get { throw null; } }
+    }
+    public delegate void UnhandledExceptionEventHandler(object sender, System.UnhandledExceptionEventArgs e);
```

Comment:
I agree this should be where it is used. The only reason we might put it somewhere else is if we need to share it, perhaps if we expose this on AppContext that might require putting this in a shared location.